moment_kinetics/src/moment_kinetics_input.jl

```diff
@@ -663,6 +511,8 @@ function mk_input(scan_input=Dict(); save_inputs_to_txt=false, ignore_MPI=true)
         error("Mirror terms not yet implemented for moment-kinetic modes")
     end
 
+    species_immutable = (ion = composition.ion, neutral = composition.neutral)
```

Comment:
Yes, we could remove species completely. I didn't do so because I wanted to make the minimal number of changes to make it easier for you to approve. Happy to wait for the merge with kinetic electrons (or do it after the merge, if it's me that does the merge here).